pythontabcmd2/parsers/create_extracts_parser.py

```diff
@@ -29,15 +29,16 @@ def create_extracts_parser():
                                                 'of embedded data source '
                                                 'names within the target'
                                                 ' workbook. ')
-        create_extract_parser.add_argument('--encrypt',
+        create_extract_parser.add_argument('--encrypt', action='store_false',
                                            help='Create encrypted extract')
         create_extract_parser.add_argument('--include-all',
+                                           action='store_false',
                                            help='Include all embedded data '
                                                 'sources within target'
                                                 ' workbook. Only available '
                                                 'when creating extracts '
                                                 'for workbook.')
-        create_extract_parser.add_argument('--project',
+        create_extract_parser.add_argument('--project', default='default',
```

Comment:
shouldn't the default for a project be None?

Reply:
Thanks for catching it. It should be None